### PR TITLE
Add notes from dweb sync + protocol labs support meeting

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

**Review by dcwalk — APPROVED** [review]
lgtm, tho I wasn't there for the first 30 mins

File: meeting_notes/20170808_dweb-sync-and-protocol-labs-support.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Protocol Labs Support
+
+- Discussed culture fit and goal alignment with Protocol Labs as an organization, and people's individual hopes and dreams from benhylau's perspective
+- knapsackforhope.org integrations (kind of what we wanted from Outernet way back...)
+- dcwalk: IPFS Cluster, possible integration points? benhylau spoke briefly with MHz about Cluster as well during Lab Day (benhylau will investigate)
+- This was brought up during our discussions: https://github.com/ipfs/docs
+
+## To Do
+
+- Figure out who is interested in assembling in November, full-time for two weeks, to iron out details about forming this working entity in 2019
```
Comment (garrying) [inline meeting_notes/20170808_dweb-sync-and-protocol-labs-support.md:86]: Not totally recalling the discussion around this, but `full-time for two weeks` seems like a significant commitment that should be made clear. Along with the expected outcomes for those involved.

**Review by garrying — APPROVED** [review]

File: meeting_notes/20170808_dweb-sync-and-protocol-labs-support.md (hunk 1 above)
Comment (benhylau) [reply meeting_notes/20170808_dweb-sync-and-protocol-labs-support.md:86]: Let's sort out availability and expectations on the hackpad?

Comment (garrying) [reply meeting_notes/20170808_dweb-sync-and-protocol-labs-support.md:86]: yeah, for sure

**Pull request merged**